recovery_backend/client/client.go

```diff
@@ -0,0 +1,65 @@
+package client
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/pkg/errors"
+	"github.com/rancher/wrangler/pkg/clients"
+	"github.com/rancher/wrangler/pkg/schemes"
+
+	v1 "k8s.io/api/apps/v1"
+	"k8s.io/client-go/informers"
+	clientset "k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+
+	"github.com/longhorn/longhorn-manager/datastore"
+	lhclientset "github.com/longhorn/longhorn-manager/k8s/pkg/client/clientset/versioned"
+	lhinformers "github.com/longhorn/longhorn-manager/k8s/pkg/client/informers/externalversions"
+)
+
+type Client struct {
+	clients.Clients
+	Datastore *datastore.DataStore
+}
+
+func New(ctx context.Context, config *rest.Config, namespace string) (*Client, error) {
```

Comment:
nit: the code is almost the same as the webhook implementation, so would suggest abstract the same part, then reuse in both places if needed.

Reply:
Addressed. Thank you.